hvpy/api.py

```diff
@@ -1,13 +1,76 @@
 __all__ = ["fake_api_call"]
 
+from datetime import datetime
+from typing import Optional
+import requests
+from pydantic import BaseModel, Field
 
-def fake_api_call():
+
+class InputParameters(BaseModel):
+    date: datetime
+    sourceId: int
+    jpip: Optional[bool] = False
+    Json: Optional[bool] = Field(False, alias="json")
```

Comment:
To fix this for this InputParameters instance, and all others, create io.py and put the following class in it
```suggestion
    class HvpyParameters(BaseModel):
        def dict(self):
            d = super().dict()
            # Pydantic doesn't allow using lower case 'json' as a field,
            # so override it here.
            if 'Json' in d:
                d['json'] = d['Json']
                del d['Json']
            return d
        def get_output_type(self):
            return OutputType.Raw
```